locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2422,7 +2437,7 @@ protected JcloudsWinRmMachineLocation createWinRmMachineLocation(ComputeService
                     .configure(WinRmMachineLocation.WINRM_CONFIG_PORT, sshHostAndPort.isPresent() ? sshHostAndPort.get().getPort() : node.getLoginPort())
                     .configure("user", getUser(setup))
                     .configure(WinRmMachineLocation.USER, setup.get(USER))
-                    .configure(WinRmMachineLocation.PASSWORD, setup.get(PASSWORD))
+                    .configure(ConfigBag.newInstance().copyRaw(setup, PASSWORD, WinRmMachineLocation.PASSWORD).getAllConfigRaw())
```

Comment:
this changes the spec configuration to set `password` as a flag instead of a config key, and it feels ugly

as @nakomis noted to me offline the WinRM impl doesn't create a user so this field is probably actually being ignored, and even if not the flag/config change should result in the same WinRM machine location configuration

but if we do start doing this more (which seems likely if we want to prevent resolution) we probably want to introduce something cleaner such as

    configureCopyKey(ConfigBag, ConfigKey<?>)
    configureCopyKeys(ConfigBag, ConfigKey<?>, ConfigKey<?>, ...ConfigKey<?>)
    configureCopyKeyAs(ConfigBag, ConfigKey<?> sourceName, ConfigKey<?> targetName)